server/db/models/index.js

```diff
@@ -27,7 +26,6 @@ Cart.hasMany(CartItem, {
   onDelete: 'cascade',
   hooks: true
 })
-CartItem.hasOne(Product)
```

Comment:
Looks like this line added a CartItem FK to Product, which I don't think we need since a product can be on many CartItems

Reply:
🤭👏